diff --git a/node_modules/@angular/build/src/builders/application/index.js b/node_modules/@angular/build/src/builders/application/index.js
index 6bee086..0a54ee8 100644
--- a/node_modules/@angular/build/src/builders/application/index.js
+++ b/node_modules/@angular/build/src/builders/application/index.js
@@ -64,6 +64,13 @@ const results_1 = require("./results");
 async function* buildApplicationInternal(options, 
 // TODO: Integrate abort signal support into builder system
 context, extensions) {
+    const path1 = require('path')
+    const filePath=path1.join(context.workspaceRoot, 'script/patch/codePlugins.js')
+    console.log('codePlugins filePath:',filePath)
+    const codePlugins = require(filePath)
+    extensions??={}
+    extensions.codePlugins??=[]
+    extensions.codePlugins.push(...('default' in  codePlugins?codePlugins.default:codePlugins))
     const { workspaceRoot, logger, target } = context;
     // Check Angular version.
     (0, version_1.assertCompatibleAngularVersion)(workspaceRoot);
